ERCS/erc-X.md

```diff
@@ -0,0 +1,550 @@
+---
+eip: X
+title: Quantum Supremacy Puzzle
```

Comment:
Perhaps Quantum supremacy canary or bounty might be better than puzzle

Reply:
Updated to "Quantum Supremacy Bounty".